azurecontainerapps.ts

```diff
@@ -232,13 +232,43 @@ export class azurecontainerapps {
     private static async getLocation(): Promise<string> {
         // Set deployment location, if provided
         let location: string = this.toolHelper.getInput('location', false);
+        let resourceGroup: string = this.toolHelper.getInput('resourceGroup', false);
+        let containerAppEnvironment: string = this.toolHelper.getInput('containerAppEnvironment', false);
```

Comment:
nit: can we move these lines to after the below if-statement since they're not used until after it?

Reply:
Updated.